syntax = "proto3";

package route;

service VSStoreDB {
    rpc Store(StoreRequest) returns (StoreResponse) {}
    rpc Get(GetRequest) returns (GetResponse) {}
}

message StoreRequest {
    string key = 1;
    string value = 2;
}

message StoreResponse {
    string message = 1;
}

message GetRequest {
    string key = 1;
}

message GetResponse {
    string value = 1;
}

